.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+# CircleCI configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2.1
+
+jobs:
+  
+  linux-python-38: &linux-template
+    docker:
+      - image: circleci/python:3.8
+
+    environment:
+      PYTHON: python
+
+    working_directory: ~/repo
+
+    steps:
+
+      - checkout
+      - run:
+          name: install dependencies
+          command: |
+            wget https://repo.anaconda.com/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh
+            bash miniconda.sh -b -p "$HOME"/miniconda
+            source /home/circleci/miniconda/etc/profile.d/conda.sh
+            conda activate base
+            conda config --set always_yes yes --set auto_update_conda false
+            conda update conda
+            conda create -n test python=$PYTHON_VERSION
+            conda activate test
+            conda install -n test --file requirements.txt
+            conda install -n test anaconda-project flake8 pillow mock pytest pytest-cov
+            export PATH=$PATH:$HOME/.local/bin
+            pip install conda
```

Comment:
@bdice @CryoZEUS I was wondering whether the sequence of the commands will matter here or not?
Shouldn't `pip install conda` be executed at first?

Reply:
We should not be running `pip install conda`. We should get conda by using a different base image, `continuumio/miniconda3`.

Reply:
Does this mean that we get conda, from the command above `pip install conda`, by using continuumio/miniconda3 and below that command, by using pip?

Reply:
Not quite. This is currently using an image `circleci/python:3.8` but we can use a different image that comes with conda already installed.

Reply:
Yeah I have started testing the `continuumio/miniconda3` on my local machine using a docker